src/TesterInternal/MockStorageProvider.cs

```diff
@@ -1,161 +0,0 @@
-﻿using System;
```

Comment:
dang it, was scratching my head thinking how would some tests compile without this class, until I noticed these were loose files not referenced by a project